CONTRIBUTION_GUIDE.md

```diff
@@ -0,0 +1,69 @@
+# Contribute to the RERUM Geolocator
+
+## Developers
+
+Bryan Haberberger -- bryan.j.haberberger@slu.edu
```

Comment:
These are appearing inline instead of on new lines.  You'll need to fix it.
![image](https://github.com/CenterForDigitalHumanities/geolocator/assets/3287006/6bce08b1-1c92-4bb0-80b4-713ca7ceb1ab)